src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -111,7 +111,9 @@ export class AssetsTable extends React.Component {
   }
 
   deleteAsset() {
-    this.props.deleteAsset(this.props.assetsParameters, this.state.assetToDelete.id);
+    this.props.deleteAsset(this.props.assetsParameters,
+      this.state.assetToDelete.id,
+      this.props.courseDetails);
```

Comment:
This line was too long for the linter - I'm not sure how I feel about this formatting either, any thoughts?

Reply:
Multi line is so hard! I’m ok with what you have here.